runsc/boot/config.go

```diff
@@ -138,6 +138,9 @@ type Config struct {
 	// Overlay is whether to wrap the root filesystem in an overlay.
 	Overlay bool
 
+	// FSGoferHostUDSAllowed enables the gofer to mount a host UDS.
+	FSGoferHostUDSAllowed bool
```

Comment:
Please also add it to Config.ToFlags(). Then this option is automatically propagated to all other commands (including `runsc gofer`)